rust-escrow/conditional-escrow/src/lib.rs

```diff
@@ -62,6 +62,15 @@ impl ConditionalEscrow {
         }
     }
 
+    pub fn proportion_deposit_of(&self, payee: &AccountId) -> Balance {
```

Comment:
Since we are passing the `payee`: 

```suggestion
    pub fn get_shares_of(&self, payee: &AccountId) -> Balance {
```